src/client/src/Domains/Messages/MessageSendForm/index.tsx

```diff
@@ -34,8 +34,8 @@ function MessageSendForm({ receiver }: MessageSendFormProps) {
     const request = React.useCallback(() => sendMessage(message, receiver), [message, receiver]);
     const [sendRequest, isLoading] = useApi(request, {
         onSuccess: () => {
-            io.emit('chat', {
-                message: message,
+            io.emit('new_message', {
+                message: message.message,
```

Comment:
Maybe a better name for the message property of the message object would be content. so message.content

Reply:
Done.